### ISSUE TITLE
Arm64 architecture chip execution error

### ISSUE THREAD

**Issue by trestea** [kubesphere/kubekey#734]
### What is version of KubeKey has the issue?

1.1.1 arm64

### What is your os environment?

麒麟v10

### A clear and concise description of what happend.

Error in executing command KK

![image](https://user-images.githubusercontent.com/17846940/139230748-f57686a2-ee96-4971-ab12-f9a7cd9f143a.png)

### Relevant log output

```shell
段错误
```

**Comment by pixiake** [comment]
Non-amd64 nodes need to use the parameter `arch` to specify the node arch.
such as:
```
- {name: node2, address: 172.16.0.3, internalAddress: 172.16.0.3, password: xxx, arch: arm64}
```

We will continue to improve the documentation.
